client/src/app/site/pages/meetings/pages/agenda/agenda.subscription.ts

```diff
@@ -0,0 +1,58 @@
+import { Observable } from 'rxjs';
+import { Id } from 'src/app/domain/definitions/key-types';
+import { ViewMeeting } from 'src/app/site/pages/meetings/view-models/view-meeting';
+
+import { pollModelRequest } from '../polls/polls.subscription';
+
+export const AGENDA_LIST_ITEM_SUBSCRIPTION = `agenda_list`;
+
+export const agendaItemFollow = [
+    {
+        idField: `list_of_speakers_id`,
+        follow: [
+            {
+                idField: `speaker_ids`,
+                follow: [
+                    {
+                        idField: `user_id`,
+                        fieldset: [
+                            `pronoun`,
+                            `first_name`,
+                            `last_name`,
+                            `username`,
+                            { templateField: `number_$` },
+                            { templateField: `structure_level_$` }
+                        ]
```

Comment:
I will add `@deprecated` to `additionalFields` to not forget about that. I think we need to be a bit cautious about that because most of the model request are not assigned to a type.